builder/lxc/config.go

```diff
@@ -11,19 +11,27 @@ import (
 	"github.com/mitchellh/mapstructure"
 )
 
+type LxcTemplateConfig struct {
+	Name       string
+	Parameters []string
+	EnvVars    []string `mapstructure:"environment_vars"`
+}
+
+type RootFsConfig struct {
+	ConfigFile string `mapstructure:"config"`
+	Archive    string
+}
+
 type Config struct {
 	common.PackerConfig `mapstructure:",squash"`
-	ConfigFile          string   `mapstructure:"config_file"`
-	OutputDir           string   `mapstructure:"output_directory"`
-	ContainerName       string   `mapstructure:"container_name"`
-	CommandWrapper      string   `mapstructure:"command_wrapper"`
-	RawInitTimeout      string   `mapstructure:"init_timeout"`
-	CloneSource         string   `mapstructure:"clone_container"`
-	CleanupFirst        bool     `mapstructure:"cleanup_first"`
-	Name                string   `mapstructure:"template_name"`
-	Parameters          []string `mapstructure:"template_parameters"`
-	EnvVars             []string `mapstructure:"template_environment_vars"`
-	TargetRunlevel      int      `mapstructure:"target_runlevel"`
+	ConfigFile          string            `mapstructure:"config_file"`
+	OutputDir           string            `mapstructure:"output_directory"`
+	ContainerName       string            `mapstructure:"container_name"`
+	CommandWrapper      string            `mapstructure:"command_wrapper"`
+	RawInitTimeout      string            `mapstructure:"init_timeout"`
+	LxcTemplate         LxcTemplateConfig `mapstructure:"lxc_template"`
+	RootFs              RootFsConfig      `mapstructure:"rootfs"`
```

Comment:
Do we need multiple side disks right off the bat, e.g. androidemu and androidsdk?

It seems that we could easily extend this by making it an array of `RootFsConfig` though.